### PR TITLE
Rewriting old data monitor text

### PR DESCRIPTION
**Pull request by jsoref**
See [JENKINS-55130 - Rewrite OldDataMonitor configure text](https://issues.jenkins-ci.org/browse/JENKINS-55130).

### Proposed changelog entries

* JENKINS-55130 - Rewrite OldDataMonitor configure text

### Submitter checklist

- [x] JIRA issue is well described
- [ ] Changelog entry appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
      * Use the `Internal: ` prefix if the change has no user-visible impact (API, test frameworks, etc.)
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

Note: I've invalidated the old localizations per @daniel-beck 's recommendations

### Desired reviewers

### CODE REVIEW

**Review by jvz — APPROVED** [review]

**Comment by jsoref** [comment]
@jvz : hi, sorry for changing the PR, this is simply an append, hopefully you're ok w/ it here, if not, I can tear it out into its own PR.

**Review by jvz — APPROVED** [review]

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties [hunk 1]
```diff
@@ -22,30 +22,31 @@
...
-  this form is not used.  Also if any unreadable data errors are reported in the right side \
-  of the table above, note that this data will be lost when the file is resaved.
+  The form below may be used to update these projects to the current format.  Doing so means a \
+  downgrade to a Jenkins release or plugin older than the selected version will not be able to \
+  read the data stored in the new format.  Note that simply using Jenkins to create and \
+  configure jobs and run builds can save data that may not be readable by older Jenkins or \
+  plugin versions, even when this form is not used.  Also if any unreadable data errors are \
+  reported in the right side of the table above, note that this data will be lost when the \
+  project is resaved.
 blurb.4=\
```
Comment (oleg-nenashev) [inline core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties]: AFAICT, it's already 150 releases behind

**Review by oleg-nenashev — APPROVED** [review]
I am not sure about removing localizations. Although the English text is improved a lot, and the comments in JENKINS-55130 are definitely right, maybe keeping the text is OK. But I am also fine with removing it

**Review by oleg-nenashev — COMMENTED** [review]
blurb2 could be retained in localizations tho. there is only a minor change

**Comment by LinuxSuRen** [comment]
Thanks for your remind. @oleg-nenashev

**Review by vilacides — APPROVED** [review]

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties [hunk 2]
```diff
@@ -22,30 +22,31 @@
 
 blurb.1=\
   When there are changes in how data is stored on disk, Jenkins uses the following strategy: \
-  data is migrated to the new structure when it is loaded, but the file is not resaved in the \
-  new format.  This allows for downgrading Jenkins if needed.  However, it can also leave data \
-  on disk in the old format indefinitely.  The table below lists files containing such data, \
-  and the Jenkins version(s) where the data structure was changed.
+  data is migrated to the new structure when it is loaded, but the project is not resaved in \
```
Comment (daniel-beck) [inline core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties]: Not necessarily correct. _Item_ might take care of, well, items (including other job types, folders, etc.), but this often applies to build records.

**Comment by jsoref** [comment]
@oleg-nenashev: yeah, that's reasonable; updated.

**Review by oleg-nenashev — APPROVED** [review]

File: core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties (hunk 2 above)
Comment (alecharp) [reply core/src/main/resources/hudson/diagnosis/OldDataMonitor/manage.properties]: I agree with @daniel-beck. `project` is too close to "job" imo.

**Review by alecharp — COMMENTED** [review]
I have a doubt about the usage of `project` word to replace `file` in different messages. 

To me, `file` was correct when `project` limit the scope of what might have change.

**Comment by oleg-nenashev** [comment]
@daniel-beck @alecharp if it is critical, could you please request changes?

**Review by alecharp — CHANGES_REQUESTED** [review]
I do believe that the replace of `files` by `projects` reduce the scope of the potential changes, which is not 100% correct imo

**Review by alecharp — APPROVED** [review]
`item` is good to me.

**Review by oleg-nenashev — APPROVED** [review]
lgtm as well

**Comment by oleg-nenashev** [comment]
Taking the feedbeck, I am going to merge it tomorrow if there is no negative feedback.

**Comment by daniel-beck** [comment]
As I wrote before,

> Not necessarily correct. _Item_ might take care of, well, items (including other job types, folders, etc.), but this often applies to build records.

This is still not addressed here.

I don't think we have an established common term for both items and builds/runs, but it would be needed here. Not to mention, potentially, agents, views, etc.

**Comment by oleg-nenashev** [comment]
> I don't think we have an established common term for both items and builds/runs

ModelObject is a Jenkins term, I believe
@daniel-beck I would appreciate if you request changes in such cases. It is hard to track comments, especially when the requests are partially addressed.

**Comment by jsoref** [comment]
I can replace `items` with `items/records`...

**Comment by jsoref** [comment]
I've replaced `items` with `items/records`...

Neither `model` nor `object` would be appropriate for use in the ui...

**Comment by alecharp** [comment]
@daniel-beck could you please see if the latest changeset is fine for you? Thanks.

**Review by daniel-beck — APPROVED** [review]

**Comment by daniel-beck** [comment]
Thanks for the update, and sorry for the delay in reviewing this!

**Pull request merged**